static/app/components/smartSearchBar/index.tsx

```diff
@@ -1324,6 +1324,7 @@ class SmartSearchBar extends React.Component<Props, State> {
         type="text"
         placeholder={placeholder}
         id="smart-search-input"
+        data-test-id="smart-search-input"
```

Comment:
I think you can query this with [`byPlaceholderText`](https://testing-library.com/docs/queries/byplaceholdertext)

Reply:
100%, I added data-test-ids on purpose.